src/pages/token-metadata/transfer.md

```diff
@@ -4,14 +4,14 @@ metaTitle: Token Metadata - Transferring Assets
 description: Learn how to transfer Assets on Token Metadata
 ---
 
-The owner of an asset can transfer it to another account by sending a **Transfer** instruction to the Token Metadata program. This instruction accepts the following attributes:
+The owner of an NFT can transfer it to another account by sending a **Transfer** instruction to the Token Metadata program. This instruction accepts the following attributes:
 
-- **Authority**: The signer that authorized the transfer. Typically, this is the owner of the asset but note that certain delegated authorities can also transfer assets on behalf of the owner as discussed in the "[Delegated Authorities](/token-metadata/delegates)" page.
-- **Token Owner**: The public key of the current owner of the asset.
-- **Destination Owner**: The public key of the new owner of the asset.
-- **Token Standard**: The standard of the asset being transferred. This instruction works for all Token Standards in order to provide a unified interface for transferring assets. That being said, it is worth noting that non-programmable assets can be transferred using the **Transfer** instruction of the SPL Token program directly.
+- **Authority**: The signer that authorized the transfer. Typically, this is the owner of the NFT but note that certain delegated authorities can also transfer assets on behalf of the owner as discussed in the "[Delegated Authorities](/token-metadata/delegates)" page.
+- **Token Owner**: The public key of the current owner of the NFT.
+- **Destination Owner**: The public key of the new owner of the NFT.
+- **Token Standard**: The standard of the NFT being transferred. This instruction works for all Token Standards in order to provide a unified interface for transferring assets. That being said, it is worth noting that non-programmable assets can be transferred using the **Transfer** instruction of the SPL Token program directly.
```

Comment:
Here is another example of how switching asset to NFT makes it less clear what is going on. The token standard can specify things that are not NFTs, so saying that token standard means the standard of an NFT is confusing.